experimental/pagerduty-webhook/program.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import requests
+import json
+import click
+
+
+@click.command()
+@click.option('--authorizationtoken', prompt='API token', help='PagerDuty API token')
+def cli(authorizationtoken):
+    """Simple program that adds a webhook to Pager Duty ."""
+    headers = getHeaders(authorizationtoken)
+    url = "https://api.pagerduty.com"
+    for key, value in get_services(headers, url).items():
```

Comment:
Can we use https://developer.pagerduty.com/api-reference/b3A6MjkyNDc4Mw-list-webhook-subscriptions to get all current subscriptions and then use list comprehension to remove existing services using the reference in the filter?

Reply:
You beat me to it before i was leaving for nursery run had a look  in postman and its doable so will modify the script to check from list of webhook subscription.